cpp/subprojects/boost.wrap

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[wrap-git]
+url=https://github.com/boostorg/boost
+revision=boost-1.83.0
+method=cmake
+clone-recursive=true
```

Comment:
This makes the initial setup pretty slow because it recurses a lot of unnecessary submodules in the boost repo. Didn't want to try and solve that now, but would be good to eventually fix

Reply:
Can we use `boost-X.Y.Z-cmake.tar.xz` instead?
For example, `boost-1.87.0-cmake.tar.xz` exists at https://github.com/boostorg/boost/releases/tag/boost-1.87.0 .